packages/commons-ui-core/src/RichTypography/index.js

```diff
@@ -0,0 +1,55 @@
+import Typography from "@mui/material/Typography";
+import { makeStyles } from "@mui/styles";
+import { PropTypes } from "prop-types";
+import * as React from "react";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    "& a": {
+      color: "red",
+    },
+  },
+}));
```

Comment:
This is the [legacy](https://mui.com/styles/basics/) way of styling components. We should move away from it. MUI has introduced a [new recommended way](https://mui.com/system/styled/) of styling moving forward which we should adopt.